### ISSUE TITLE
About dialog of Red console is odd

### ISSUE THREAD

**Issue by lkppo** [red/red#3630]
The about dialog of Red console it odd. It seems the labels have a grey / white color on a black background.

Tested with red-03dec18-31c0a32a.exe on Windows 7 64 bits.

**Comment by greggirwin** [comment]
Is it the same on older builds? Any special display settings? Can you post a screenshot?

**Comment by endo64** [comment]
Looks normal on W10.

**Comment by lkppo** [comment]
Hi Gregg. It's working on Windows 10, good, but on Windows 7 it's not working. What to add? I can't post an image because my browser version is to old for github. I will post it on the list.

**Comment by lkppo** [comment]
Le 09/12/2018 à 20:23, Gregg Irwin a écrit :
> Failing any new comments from @lkppo <https://github.com/lkppo>, I'm 
> planning to close this ticket.

Here is what to illustrate the bug #3630.

-- 
Stéphane Aulery

**Comment by greggirwin** [comment]
Starting with the following, assuming it shows the problem for you, tweak it and see if you can make the problem go away, e.g., change style colors, include alpha value, eliminate styles. What is the most basic thing that shows the problem.
```
	view [
		title "About"
		backdrop 58.58.60

		style text:  text 360 center 58.58.60
		style txt:   text font-color white
		style small: txt  font [size: 9 color: white]

		below
		txt bold "Red Programming Language" font [size: 15 color: white]
		small 360x20 "Copyright 2011-2018 - Red Foundation"
		button "Close" [unview]
	]
```

**Comment by lkppo** [comment]
Your code shows the bug. I will try debug it. Thanks.

-- 
Stéphane Aulery

**Comment by lkppo** [comment]
Le 09/12/2018 à 21:17, Gregg Irwin a écrit :
> Starting with the following, assuming it shows the problem for you, 
> tweak it and see if you can make the problem go away, e.g., change style 
> colors, include alpha value, eliminate styles. What is the most basic 
> thing that shows the problem.
> 
> |view [ title "About" backdrop 58.58.60 style text: text 360 center 
> 58.58.60 style txt: text font-color white style small: txt font [size: 9 
> color: white] below txt bold "Red Programming Language" font [size: 15 
> color: white] small 360x20 "Copyright 2011-2018 - Red Foundation" button 
> "Close" [unview] ] |

If "backdrop 58.58.60" is commented, the background is consistent.

Changing alphp do nothing.

It seems "style text:  text 360 center 58.58.60" doesn't change the 
background color. "style text:  text 360 center" give the same output.

So I did "style text:  field 360 center 58.58.60 no-border" and it's worked:

view [
	title "About"
	backdrop 58.58.60

	style text:  field 360 center 58.58.60 no-border
	style txt:   text font-color white
	style small: txt  font [size: 9 color: white]

	below
	txt bold "Red Programming Language" font [size: 15 color: white]
	small 360x20 "Copyright 2011-2018 - Red Foundation"
	button "Close" [unview]
]

Can text face have a background?

-- 
Stéphane Aulery

**Comment by greggirwin** [comment]
@endo64 can you follow up on this, to make sure @qtxie has what he needs to comment or fix?

**Comment by qtxie** [comment]
@lkppo Yes. `Text` face has a background. Does it work for you? The background of the "def" should be red.
```
view [style text2: text 222.0.0 text "abc" text2 "def"]
```

**Comment by lkppo** [comment]
Le 13/12/2018 à 12:57, Qingtian a écrit :
> @lkppo <https://github.com/lkppo> Yes. |Text| face has a background. 
> Does it work for you? The background of the "def" should be red.
> 
> |view [style text2: text 222.0.0 text "abc" text2 "def"] |

It's not working. I see black texts on the default background color (grey).

-- 
Stéphane Aulery

**Comment by endo64** [comment]
@qtxie @lkppo I see red background on Win10, I have no Win7 installed so cannot test on it.

**Comment by qtxie** [comment]
@lkppo Are you using the default Aero theme on Win7?

**Comment by lkppo** [comment]
Le 15/12/2018 à 01:12, Qingtian a écrit :
> @lkppo <https://github.com/lkppo> Are you using the default Aero theme 
> on Win7?

No, I switched to defaults (Aero) and It's not working better.

-- 
Stéphane Aulery

**Comment by lkppo** [comment]
New phenomenon.

I just downloaded the latest version (red-14dec18-25777f5e.exe) and at 
the end of compilation the console is displayed. I immediately opened 
the dialog and it was correct.

I closed the console, then I reopened it. The dialog was yet false.

Looking more closely I realize that if I run a version 0.6.3 there is an 
error, while the 0.6.4 is correct.

In fact I run the console with an icon pinned to the task bar. He did 
not pin the downloaded version but the compiled version which explains 
the inconsistency.

I think I made you look for nothing. Sorry.

-- 
Stéphane Aulery
